contracts/hackatom/src/msg.rs

```diff
@@ -83,6 +86,20 @@ pub enum QueryMsg {
     GetInt {},
 }
 
+// TODO: this should really be generated by a nice derive macro
+impl QueryResponses for QueryMsg {
```

Comment:
You could add `#[cfg(not(target_arch = "wasm32"))]`. Then this implementation does not need to be compiled and optimized out again.

Reply:
Done!